setup/20-fty-compat.sh

```diff
@@ -183,7 +183,7 @@ if [[ -e /var/lib/bios/agent-alerts-list/state_file ]]; then
     /usr/bin/fty-alert-list-convert \
         state_file \
         /var/lib/bios/agent-alerts-list \
-        /var/lib/fty/fty-alert-list && \
-    chown bios:bios-infra /var/lib/fty/fty-alert-list/state_file && \
+        /var/lib/fty/fty-alert-list
+    chown bios:bios-infra /var/lib/fty/fty-alert-list/state_file || :
```

Comment:
Why isn't it an error that as a result of the conversion the new file does not exist?